src/app/components/work-item-quick-add/work-item-quick-add.component.less

```diff
@@ -247,7 +248,8 @@
     margin-bottom: 0;
     margin-left: 0;
     background: @color-pf-white;
-    .Border(2px, solid, @color-pf-black-400);
+    border: 2px solid;
+    border-color: @color-pf-black-400;
```

Comment:
Can't we write it in one line  ``` border: 2px solid @color-pf-black-400 ``` instead of writing it in 2 lines.

Reply:
No, Its as per guidelines only

Reply:
as per patternfly guidline?

Reply:
https://github.com/fabric8-ui/fabric8-ux/blob/master/code-guidelines.md